WebApi/Controllers/CryptoPriceController.cs

```diff
@@ -41,5 +41,20 @@ public async Task<JToken> GetBitCoinPriceAsync()
             var price = pricesPerCurrency[this.cryptoConfig.Value.Currency][this.cryptoConfig.Value.TimeSelection];
             return price;
         }
+
+        /// <summary>
+        /// Get the price of bitcoin
```

Comment:
get the price of bitcoin for for eth????? what is this bullshit? I wasn't able to understand this code because of this comment